### DISCUSSION TITLE
Inactive Meetings and the Meeting Guide

### DISCUSSION THREAD

**Discussion by mikepoconnor4** [code4recovery/12-step-meeting-list#798] [Q&A] [answered]
We have quite a few Location Temporarily Closed / Inactive meetings waiting to reopen. On our website, we filter those out by adding "?type=active" in the URL of the meeting list page. 

Is there anyway that we can not feed those up to the Meeting Guide app? We use the TSML

Is there anyway that we can not feed those up to the Meeting Guide app? We use the TSML UI to man UI to manage our meetings (as opposed to feeding a file of meetings up).

Our feeling is that showing the inactive meetings isn't doing anything for the person seeking a meeting - and just takes up space on the display in the Meeting Guide app.

**Comment by joshreisner** [comment, marked as answer]
Hi Mike, I agree. Leaving the meetings listed is a service to groups but does not help members. 

I manage the list in San Jose, CA and we decided in September 2021 to remove inactive meetings from the schedule. Just this past Sunday I ran into my first person to say that he "wishes we had contacted the group" about the change, because they were intending to come back, but in my mind I still think we did the right thing. His group is on the schedule again now that it is meeting, I don't see what harm was done, if any, by taking it down for a few months.

So my recommendation would be to take these meetings down in WordPress, either by saving as Draft or Moving to Trash. They're not hard to re-instate when needed.

Other idea: request GSO stop displaying inactive meetings. I'm not sure if this is something they want to do.

I don't know of a technical way currently to handle this in TSML. It is possible to adjust your feed URL to display meetings by type, but we don't currently have a way to filter by active meetings.

Only shows "Open" meetings:
```
https://mywebsite.org/wp-admin?admin-ajax.php?action=meetings&key=abc123&type=0
```

If we did have a way, or if we added that, then you'd be able to provide GSO with that URL instead of the current one to use for importing.

**Comment by mikepoconnor4** [comment]
Is the best place to contact the Meeting Guide developers in TIAA?

**Reply by kiyote33** [reply]
Hi Mike,

There's a Meeting Guide support page link here: [https://meetingguide.helpdocs.io](https://meetingguide.helpdocs.io) which is provided on GSO's aa.org website.  You might find the answer you are looking for there.
